webknossos/webknossos/cli/merge_fallback.py

```diff
@@ -0,0 +1,200 @@
+"""This module merges a volume annotation layer with its fallback layer."""
+
+import logging
+from argparse import Namespace
+from multiprocessing import cpu_count
+from typing import Any, List, Optional, Tuple
+
+import typer
+from cluster_tools import Executor
+from rich.progress import track
+from typing_extensions import Annotated
+
+from ..annotation import Annotation
+from ..dataset import Dataset, MagView
+from ..geometry import BoundingBox, Mag
+from ..utils import get_executor_for_args
+from ._utils import DistributionStrategy, parse_path
+
+
+def main(
+    *,
+    target: Annotated[
+        Any,
+        typer.Argument(
+            help="Path to your WEBKNOSSOS output dataset.",
+            show_default=False,
+            parser=parse_path,
+        ),
+    ],
+    source_annotation: Annotated[
+        Any,
+        typer.Argument(
+            help="Path to your WEBKNOSSOS zip annotation",
+            show_default=False,
+            parser=parse_path,
+        ),
+    ],
+    dataset_folder: Annotated[
```

Comment:
```suggestion
    dataset_path: Annotated[
```
(We reserve the word folder for the virtual folder structure in the webknossos dashboard UI. Any on-disk paths are called directory/path)

Reply:
changed the variable name accordingly.